src/platform/packages/private/kbn-grid-layout/grid/use_grid_layout_events/sensors/index.ts

```diff
@@ -7,24 +7,22 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import { GridLayoutStateManager } from '../../types';
 import { UserInteractionEvent } from '../types';
-import { isMouseEvent } from './mouse';
-import { isTouchEvent } from './touch';
+import { isKeyboardEvent, getFocusedElementPosition } from './keyboard';
+import { isMouseEvent, getMouseSensorPosition } from './mouse';
+import { isTouchEvent, getTouchSensorPosition } from './touch';
 
 export { isMouseEvent, startMouseInteraction } from './mouse';
 export { isTouchEvent, startTouchInteraction } from './touch';
+export { isKeyboardEvent, startKeyboardInteraction } from './keyboard';
 
-export function getPointerPosition(e: UserInteractionEvent) {
-  if (!isMouseEvent(e) && !isTouchEvent(e)) {
-    throw new Error('Invalid event type');
+export function getSensorPosition(e: UserInteractionEvent) {
+  if (isMouseEvent(e)) {
+    return getMouseSensorPosition(e);
+  } else if (isTouchEvent(e)) {
+    return getTouchSensorPosition(e);
+  } else if (isKeyboardEvent(e)) {
+    return getFocusedElementPosition();
   }
-  return isTouchEvent(e) ? e.touches[0] : e;
+  throw new Error('Invalid event type');
 }
-
```

Comment:
I moved it from here to `state_manager_selectors` since the idea is that sensors are agnostic about the state_manager.